CodeSniffer/Standards/PEAR/Sniffs/NamingConventions/ValidClassNameSniff.php

```diff
@@ -42,6 +42,7 @@ public function register()
         return array(
                 T_CLASS,
                 T_INTERFACE,
+                T_TRAIT,
```

Comment:
Nor me to be fair. What about if I make it a toggleable option? I.e. add a public property `checkTraits` which defaults to `false` (old behaviour), but can be changed via a custom ruleset ?